### ISSUE TITLE
Plaid Transfer API support

### ISSUE THREAD

**Issue by Jdyn** [tylerwray/elixir-plaid#12]
Hey, great work.

Would you be interested in adding support for the [Transfer API](https://plaid.com/docs/api/products/transfer/)?

**Comment by tylerwray** [comment]
Yes! I'll try and find some time to revisit the Plaid API's later this month. PR's are also welcome if you have time first.

**Comment by ktayah** [comment]
I'm actually working on this now since I needed some Transfer API functionality for my usage of this library. I'll try and get a PR soon with these changes.
